test/1.2/events/T_dup.py

```diff
@@ -4,15 +4,23 @@
 import sys
 from os.path import join
 import subprocess
+from simicsutils.host import is_windows
 
 cpu = SIM_create_object("clock", "clock", [["freq_mhz", 1]])
 obj.queue = cpu
 obj.postall = 1
 
 SIM_write_configuration_to_file("dup.chkp", Sim_Save_Nobundle)
 
+if os.path.isdir(sys.argv[0]):
+    exe_sfx = '.exe' if is_windows() else ""
+    app = [join(sys.argv[0], '..', 'mini-python') + exe_sfx, sys.argv[0]]
+else:
+    app = [sys.argv[0]]
```

Comment:
maybe even easier to drop argv and just run `f'{conf.sim.project}/bin/simics'`?

Reply:
Fixed